apps/ui/src/components/FormNetwork.vue

```diff
@@ -20,7 +20,7 @@ const availableNetworks = enabledReadWriteNetworks.map(id => {
 
 <template>
   <UiContainerSettings :title="title" :description="description">
-    <div class="grid grid-cols-auto gap-3 mb-3">
```

Comment:
By adding `space-y-3` to UiStepper, line 60, we could avoid this class

Reply:
Are you referring to `grid grid-cols-auto gap-3` ? this doesn't work with UIStepper change, these are grandchildren of UIstepper

Reply:
just the `mb-3`, for the spacing between the content and the NEXT button

Reply:
ah ok ok next button is not related this.